Mocks/Http/LeakingContainer.cs

```diff
@@ -0,0 +1,59 @@
+﻿// ==++==
+//
+//   Copyright (c) Shadowsoft Corporation.  All rights reserved.
+//
+// ==--==
+
+using System.Collections;
+using System.Diagnostics.CodeAnalysis;
+using Yextly.Common;
+
+namespace Yextly.Testing.Mocks.Http
+{
+    internal sealed class LeakingContainer : IDisposableProducerConsumerCollection<IDisposable>

```

Comment:
Do we really need this?

Reply:
Unfortunately we cannot use a `Bag<T>` since there is no `IBag<T>`